apps/package.json

```diff
@@ -43,7 +43,7 @@
     "@code-dot-org/blockly": "3.1.7",
     "@code-dot-org/bramble": "0.1.26",
     "@code-dot-org/craft": "0.1.18",
-    "@code-dot-org/dance-party": "code-dot-org/dance-party#38dd2f9f8285bf6a87454a59a39a3aab14a68931",
+    "@code-dot-org/dance-party": "code-dot-org/dance-party#d14fc65",
```

Comment:
Why not update through the PR merge commit 8342c7ec0a6140c09e3b15cd26e47b1416b2d056 to get previous fixes in the master branch?